tests/integration/test_api_gateway.py

```diff
@@ -360,7 +362,79 @@ def test_api_gateway_authorizer_crud(self):
             Exception,
             apig.get_authorizer,
             self.TEST_API_GATEWAY_ID,
-            authorizer_id)
+            authorizer_id
+        )
+
+    def test_apigateway_with_lambda_integration(self):
+        self.create_lambda_function(TEST_API_GATEWAY_INTEGRATION_LAMBDA_FUNCTION_NAME)
```

Comment:
I think in the future we should use generated resource names, rather than using hardcoded global names like `TEST_API_GATEWAY_INTEGRATION_LAMBDA_FUNCTION_NAME`, e.g.:
```
lambda_name = 'test-apigw-%s' % short_uid()
self.create_lambda_function(lambda_name)
...
```

Shouldn't hold back the merge of this PR, but something to consider in the future..